lib/validataion/validators.js

```diff
@@ -30,49 +30,58 @@ function throwError(text) {
 
 const getElementPropName = (mapOfNames, name) => typeof mapOfNames[name] === 'string' ? mapOfNames[name] : name;
 
+function prettifyElementPropsErrors(validate) {
+	const getDataName = path(['data', 'name']);
+
+	return validate.errors
+		.filter(getDataName)
+		.map(i => {
+			const dataName = getDataName(i);
+
+			if (dataName in ELEMENT_PROP_M) {
+				const elementPropTye = typeof dataName === 'string' ?
+					ELEMENT_PROP_TYPES[ELEMENT_PROP_M[dataName]] :
+					ELEMENT_PROP_TYPES[dataName];
+				const elementProp = getElementPropName(ELEMENT_PROP_M, dataName);
+
+				return `Element property '${elementProp}' should be of type ${elementPropTye}`;
```

Comment:
`elementPropTye ` - typo in constant name and why not move it to `texts`

Reply:
I dont noticed it, but I will fix it.